frappe/website/doctype/web_form/templates/web_form.html

```diff
@@ -82,7 +82,9 @@ <h3>{{ _("Comments") }}</h3>
 	time_zone: {
 		system: "{{ frappe.utils.get_time_zone() }}",
 		user: "{{ frappe.db.get_value('User', frappe.session.user, 'time_zone') or frappe.utils.get_time_zone() }}"
-	}
+	},
+	link_title_doctypes: `{{ frappe.get_all('DocType', {'show_title_field_in_link': 1}, pluck='name') +
+		frappe.get_all('Property Setter', {'property': 'show_title_field_in_link', 'value': '1'},pluck='doc_type') }}`
```

Comment:
```suggestion
	},
	link_title_doctypes: `{{ frappe.call('frappe.boot.get_link_title_doctypes') }}`
```

Can you check if this also fixes the issue? We can safely whitelist `get_link_title_doctypes`